.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.iconTheme": "eq-material-theme-icons-darker"
```

Comment:
is there a reason we're committing this file? seems like everyone could have different vscode settings

Reply:
i forgot to add it to the git ignore. I'll remove it.

Reply:
bump